scripts/update-apk.sh

```diff
@@ -4,10 +4,7 @@ set -e
 git config --global user.name "Travis CI"
 git config --global user.email "noreply+travis@fossasia.org"
 
-export DEPLOY_BRANCH=${DEPLOY_BRANCH:-development}
-export PUBLISH_BRANCH=${PUBLISH_BRANCH:-master}
-
-if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/open-event-attendee-android" ] || ! [ "$TRAVIS_BRANCH" == "$DEPLOY_BRANCH" -o "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
+if ! [[ ( $TRAVIS_PULL_REQUEST =~ ^(false)$ && $TRAVIS_REPO_SLUG =~ ^(fossasia/open-event-attendee-android)$ && $TRAVIS_BRANCH =~ ^(development|master)$ ) || ( $TRAVIS_BRANCH =~ ^(master)$ && $TRAVIS_PULL_REQUEST_BRANCH =~ ^(development)$ && $TRAVIS_REPO_SLUG =~ ^('fossasia/open-event-attendee-android')$ && $TRAVIS_PULL_REQUEST_SLUG =~ ^('fossasia/open-event-attendee-android')$ ) ]]; then
```

Comment:
Too complex, I'm just a human, not a compiler, I need readable code

Define a variable. 

`export DRY_RUN = IS_PR_TO_MASTER`

Then OR it with the if condition

Reply:
Oh yeah, better xD